depmapomics/igv.py

```diff
@@ -12,11 +15,7 @@ def get_gcloud_auth_token():
 
 
 def load_sample_tracker():
```

Comment:
you can replacce the full function

Reply:
this function adds a few extra columns with hg19_bam/hg38_bam so shouldn't be dropped as it is